backend/app/services/context/context_service.py

```
@@ -964,6 +965,282 @@ def get_unlinked_contexts(
+                # Update the binary data
+                metadata = {
+                    "filename": context.name,
+                    "mime_type": "application/vnd.openxmlformats-officedocument.presentationml.presentation",
+                    "file_size": len(pptx_binary),
+                    "user_id": context.user_id,
+                    "is_encrypted": is_encrypted,
+                }
+                storage_backend.save(storage_key, data_to_store, metadata)
+
+        # Handle preview images
+        old_preview_ids = type_data.get("preview_images", [])
```

Ruff (0.14.13)

Local variable `old_preview_ids` is assigned to but never used

Remove assignment to unused variable `old_preview_ids`

(F841)